agi_plz_ortschaften_pub/build.gradle

```diff
@@ -53,15 +66,42 @@ task dbImport(type: Ili2pgImport, dependsOn: 'unzipData'){
     strokeArcs = true
 }
 
+task convertDataEdit(type: SqlExecutor, dependsOn: 'dbImportSogis'){
+    description = "Convert plz_ortschaften-Data to Edit Schema"
+    doLast {
+        println "plz_ortschaften-Data are converted."
+    }
+	database = [dbUriEdit, dbUserEdit, dbPwdEdit]
+    sqlFiles = [
+            'delete_ortschaft.sql',
+            'delete_postleitzahl.sql',
+            'insert_ortschaft.sql',
+            'insert_postleitzahl.sql',
+	]
+}
 
-task convertData(type: Db2Db, dependsOn: 'dbImport'){
-    description = "Convert plz_ortschaften-Data"
+task convertDataSogis(type: SqlExecutor, dependsOn: 'convertDataEdit'){
+    description = "Convert plz_ortschaften-Data to Sogis Schema"
     doLast {
         println "plz_ortschaften-Data are converted."
+    }
+	database = [dbUriSogis, dbUserSogis, dbPwdSogis]
+    sqlFiles = [
+            'delete_ortschaft.sql',
+            'delete_postleitzahl.sql',
+            'insert_ortschaft.sql',
+            'insert_postleitzahl.sql',
+	]
+}
+
+task copyDataPub(type: Db2Db, dependsOn: 'convertDataSogis'){
+    description = "Copy plz_ortschaften-Data to Pub Schema"
+    doLast {
+        println "plz_ortschaften-Data are copied."
     }
     finalizedBy 'removeFiles'
 
-    sourceDb = [dbUriSogis, dbUserSogis, dbPwdSogis]
+    sourceDb = [dbUriEdit, dbUserEdit, dbPwdEdit]
     targetDb = [dbUriPub, dbUserPub, dbPwdPub]
     transferSets = [
             new TransferSet('plzortschaften_ortschaft.sql', 'agi_plz_ortschaften_pub.plzortschaften_ortschaft', true),
```

Comment:
Die Skripts _plzortschaften_ortschaft.sql_ und _plzortschaften_postleitzahl.sql_ musst du noch neu schreiben. Sie müssen nun ja keinen Datenumbau mehr machen, weil der ja bereits im Task _convertData_ gemacht worden ist. Diese beiden Skripts müssen mehr oder weniger nur noch `SELECT * FROM agi_plz_ortschaften_pub.plzortschaften_ortschaft;` (das andere Skript analog) machen, sie sollen die Daten also 1:1 in die pub-DB kopieren. (Ersetze `*` aber durch die richtigen Spaltennamen.)